--- data/dv/intermediate/docbook/HDO/Ö367-01_1.xml	2010-11-09 10:40:42.306862131 -0800
+++ tmp/tmpZQx2xY	2010-11-09 10:40:11.278861000 -0800
@@ -305,7 +305,7 @@ Anonymiserat bolagsnamn
 sexuella trakasserier.
 
 C. C.: Han är ägare till S. Via kommunen fick J. Å. en praktikantplats
-på Swerotec. Hennes arbetsuppgifter var bl.a. att svara i telefon och
+på S. Hennes arbetsuppgifter var bl.a. att svara i telefon och
 arbeta med ekonomiska uppgifter. Hon skulle läras upp och tanken var att
 det skulle kunna leda till en fast anställning. Efter fjorton dagar
 ringde de från kommunen och sade att J. Å. inte trivdes. Han frågade
