--- data/sfs/intermediate/2008/263.txt
+++ var/folders/sy/r4f01ykx7v31d8_0nmvsyc2r0000gn/T/tmpY2Xul7
@@ -758,7 +758,7 @@ Ändrat felaktig datumformattering
 
 Överklagande
 
-54 § /Upphör att gälla U:2014-081-01/
+54 § /Upphör att gälla U:2014-08-01/
 Beslut om föreläggande vid vite enligt 16 § andra stycket
 och 52 § andra stycket, begäran enligt 53 § som har förenats
 med vite samt om förseningsavgift får överklagas hos allmän
