--- data/dv/intermediate/docbook/ADO/2005-124.xml	2010-07-01 11:51:34.353000000 +0200
+++ users\staffan\appdata\local\temp\tmp98a06k	2010-07-01 11:51:00.750000000 +0200
@@ -275,9 +275,9 @@ Anonymisering av firmanamn och plats.
 månadslön var 9 040 kr.
 
 Nordiska Folkhögskolan har arrangerat ett antal korta kurser förlagda
-till Kulturbrottet på Bohus-Malmön. Kulturbrottet drivs som B.A:s
+till K. på B-M. K. drivs som B.A:s
 enskilda firma. Nordiska Folkhögskolan köper inkvartering, dvs.
-kurslokal och boende för sina kursdeltagare av Kulturbrottet. Kurserna
+kurslokal och boende för sina kursdeltagare av K. Kurserna
 har varit en del av kortkursutbudet vid Nordiska Folkhögskolan och
 företrädesvis avsett estetiska kurser eller kurser med anknytning till
 bohuslänsk kultur och/eller natur. Kurserna har administrerats och
@@ -286,7 +286,7 @@
 
 S.C. är rektor vid Nordiska Folkhögskolan sedan den 1 juli 1994. Han och
 B.A. inledde ett förhållande under våren 1988. S.C. var då rektor vid
-Skinnskattebergs Folkhögskola där även B.A. var anställd.
+S.F. där även B.A. var anställd.
 
 B.A. har fram till och med september 2002 inte upplevt några problem i
 sin anställning på Nordiska Folkhögskolan. Inga former av kritik eller
@@ -406,7 +406,7 @@
 konferensverksamhet. Det finns cirka 250 elever på de långa kurserna.
 Verksamheten bedrivs i huvudsak i egna lokaler i Kungälv. Under senare
 tid har Nordiska Folkhögskolan haft kurser förlagda även till B.A:s
-verksamhet Kulturbrottet på Bohus-Malmön.
+verksamhet K. på B-M.
 
 Det är riktigt att B.A. anställdes år 1995. Hon fick då en tidsbegränsad
 deltidsanställning. Anställningen övergick senare i en
@@ -438,9 +438,9 @@
 från S.C., fattade ledningen för Nordiska Folkhögskolan beslut om att
 ingå en överenskommelse med B.A. om att under perioden den 26 februari
 2003 till den 31 december 2007 garantera B.A. kortkurser förlagda till
-hennes anläggning Kulturbrottet på Bohus-Malmön. B.A. tilläts också att
+hennes anläggning K. på B-M. B.A. tilläts också att
 välja om hon ville fullgöra sin tjänstgöring helt eller delvis på
-Kulturbrottet, dock med den begränsningen att viss del måste förläggas
+K., dock med den begränsningen att viss del måste förläggas
 till Kungälv. Skälet till att överenskommelsen skulle gälla till och med
 år 2007 var att S.C. skulle gå i pension därefter.
 
@@ -529,7 +529,7 @@
 stället per e-post den 28 augusti 2003 till K.L. en ensidigt gjord
 planering över hur hon tänkt sig sin tjänstgöring för läsåret 2003/2004.
 B.A. ansåg sig bl.a. ha rätt att förlägga all sin tjänstgöring till
-Bohus-Malmön. Den planeringen stämde inte alls med den som beslutats för
+B-M. Den planeringen stämde inte alls med den som beslutats för
 henne under de nämnda planeringsdagarna. B.A. upprepade i samma
 e-postmeddelande dessutom en mängd kränkande omdömen om S.C. samt
 framförde att hon inte ansåg sig skyldig att diskutera sin tjänstgöring
@@ -697,7 +697,7 @@
 den 12 februari 2004 ett fax till henne om att hon omgående skulle
 infinna sig på skolan för att ta emot avskedandehandlingen. Den 16
 februari hade hon ännu inte infunnit sig varför skolans vaktmästare
-ombads att med bil åka till B.A:s bostad på Bohus-Malmön för att
+ombads att med bil åka till B.A:s bostad på B-M. för att
 överlämna avskedandebeskedet personligen till henne. Eftersom B.A. inte
 öppnade eller inte var hemma så lämnades beskedet i ett slutet kuvert i
 hennes brevlåda.
@@ -918,7 +918,7 @@
 verksamhet och för att ha nyttjat skolans materiel, t.ex. papper, kuvert
 och kopieringsmaskin, för privat bruk. Hon uppger vidare i ett av breven
 att S.C. redan som rektor på Malungs folkhögskola utnyttjade skolan för
-privata intressen, och att han på Skinnskatteberg upprepade sitt
+privata intressen, och att han på S.F. upprepade sitt
 maktmissbruk och sina sexuella trakasserier. B.A. skriver bl.a. ”. . .
 det finns inget förtroende som han inte har svikit eller är beredd att
 göra. Jag är innerligt glad att inte längre vara soptunna. Nu raggar han
